src/uct/ib/rc/accel/rc_mlx5_common.h

```diff
@@ -470,11 +470,38 @@ void uct_rc_mlx5_iface_common_sync_cqs_ci(uct_rc_mlx5_iface_common_t *iface,
 int uct_rc_mlx5_iface_commom_clean(uct_ib_mlx5_cq_t *mlx5_cq,
                                    uct_ib_mlx5_srq_t *srq, uint32_t qpn);
 
+#if IBV_HW_TM
+void uct_rc_mlx5_init_rx_tm_common(uct_rc_mlx5_iface_common_t *iface,
+                                   unsigned rndv_hdr_len);
+
 ucs_status_t uct_rc_mlx5_init_rx_tm(uct_rc_mlx5_iface_common_t *iface,
                                     const uct_rc_iface_common_config_t *config,
                                     struct ibv_exp_create_srq_attr *srq_init_attr,
-                                    unsigned rndv_hdr_len,
-                                    unsigned max_cancel_sync_ops);
+                                    unsigned rndv_hdr_len);
+#else
+static UCS_F_MAYBE_UNUSED ucs_status_t
```

Comment:
where is uct_rc_mlx5_init_rx_tm() used without IBV_HW_TM?

Reply:
in uct_rc_mlx5_init_rx()